bundles/dip/Citrix/com.sapho.services.concur.ConcurService/0.7.7/i18n/ja.json

```diff
@@ -1,173 +1,173 @@
-    "813170e4-d5b8-4dff-a3ca-2bd9ec095167" : "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
-    "5abaa283-a651-418d-8669-5bf96565d5ba" : "提出日",
-    "3316c787-2fd6-42f0-983f-26a84934cb6c" : "Checkbox",
-    "2bdb055d-58df-4264-b163-6a5d80493236" : "総額",
-    "f4ea29d7-d2da-4c50-8a4f-e8286c6c6517" : "経費の詳細",
-    "6972a08a-a606-457b-a418-4f64036f4d11" : "Expense Report Details",
-    "00d459fc-e8b1-4b59-927c-9a963cc9082f" : "表示するデータがありません",
-    "b48fbbef-e94b-4637-b213-88b9fa18918d" : "レポート番号",
-    "31aae848-55b8-4410-9d3d-ea8a79e07a20" : "承認者",
-    "eb48dd0e-088e-4240-8d15-95b8a8b2d985" : "ベンダーの説明"
+  "7f261cc3-8110-44e7-9284-e925a4b72c00": {
+    "d54f58a3-466f-4d94-898a-09d219c7696c": "承認者",
+    "e00f4bc8-36e1-4d8a-b690-399ee5b16877": "自分の保留中の経費レポート",
+    "571d160f-e584-4d60-aff3-419766ea4fea": "ベンダー",
+    "3d8b658b-c26d-46e9-972f-bacdcfa72e93": "Your {{format total \"decimal\"}} {{currency_code}} expense report for \"{{name}}\" was sent back.",
+    "22aafde8-ed2d-4f42-97df-ff8900975675": "提出日",
+    "8c1d3714-eae7-461a-bc43-6cade1f840c8": "Your {{format total \"decimal\"}} {{currency_code}} expense report for \"{{name}}\" was approved.",
+    "21992559-20a4-4716-aa8b-f0166a8ebb8f": "日付",
+    "37cbfc59-da56-4ec3-98ca-87010bceb9d5": "経費",
+    "6b3b7b51-b930-4c37-ae7a-3c8fea9043ab": "経費レポートの詳細",
+    "883756e8-41e2-414a-af49-6a85ca876b9c": "レポート",
+    "ae846bf2-c332-4a13-9689-e7cc4a5f37ba": "総額",
+    "c602650c-b691-4374-b44b-ef56eef32fd1": "支出カテゴリ",
+    "44dc30c3-f3f8-437d-abdf-dc4202abd208": "取引金額",
+    "28e10e8e-22d8-4458-99b8-2a7faf343bc4": "提出した経費レポートを検索して表示します。 ",
+    "717f16ce-ede5-4146-9b82-480d4e15495c": "申請の差戻し",
+    "945d0fea-5d3a-4e4e-9345-232ec0eac8c8": "経費ポリシー",
+    "ee1cc89f-7a71-4b7f-bbb7-0cfd1530336b": "申請が承認されました",
+    "23df46cb-8113-4bcf-bcee-82fbd462562e": "ステータス",
+    "38a5df72-9798-42e5-b9eb-418d8705165e": "表示するデータがありません",
+    "81f45b31-9e4a-46ce-9ae3-137858d2f5a7": "自分の保留中の経費レポート",
+    "ca4b06fe-9fd4-41f4-b272-25f5ecfa8df3": "Expense Details",
+    "2b5cf778-50d4-4fa5-9ccc-a502478aca32": "領収書",
+    "9c3cb4ad-23b1-4ae8-b06e-87718350e6fe": "レポート名",
+    "a608be54-ffb8-40c9-a2c8-89e7f09e651d": "レポートの差戻し",
+    "b72b782f-21a2-47eb-bc26-07caaa3555b1": "レポートが承認されました",
+    "5b5c624a-c8e6-48ca-b248-8138fe392d14": "日付",
+    "16216c35-ea37-406c-8333-d01135a08117": "タイプ",
+    "c14873a4-0ac1-4c32-b5e1-bcbaeafb0f14": "{{format total \"decimal\"}} {{currency_code}}",
+    "6702460c-5461-4d9a-8eb3-a150f55d50bc": "取引金額",
+    "308e9f76-f2dc-4cf4-870b-6676c917b99f": "レポートの検索",
+    "813170e4-d5b8-4dff-a3ca-2bd9ec095167": "{{format transaction_amount \"decimal\"}} {{transaction_currency_code}}",
+    "5abaa283-a651-418d-8669-5bf96565d5ba": "提出日",
+    "3316c787-2fd6-42f0-983f-26a84934cb6c": "チェックボックス",
+    "2bdb055d-58df-4264-b163-6a5d80493236": "総額",
+    "f4ea29d7-d2da-4c50-8a4f-e8286c6c6517": "経費の詳細",
+    "6972a08a-a606-457b-a418-4f64036f4d11": "Expense Report Details",
```

Comment:
Why not translate this string?

Reply:
@yiminl-ctx  this PR is just to sync the files to fix the g11n scanner issue. translations will be followed.